### ISSUE TITLE
internet connection suffers after starting GMR

### ISSUE THREAD

**Issue by beebase** [derekhe/msfs2020-map-enhancement#78]
I have a 1Gbit network connection with 900 + up/download speed.
Once I start running GMR, the speed goes down dramatically to a point that my browser won't react anymore.
It's weird because GMR doesn't occupy more than 5 Mbps on average.
As a starting point can anyone test their connection with this tool while GMR is running?

https://www.microsoft.com/store/productId/9NBLGGH4Z1JC

Trying to do this speedtest while GMR is running will consequently fail on my machine.

**Comment by beebase** [comment]
> Map service provider restriction

How can a map service provider kill my entire internet connection?

**Comment by derekhe** [comment]
One guy has reported this issue before, I can't do any thing for ISP restrions.

**Comment by beebase** [comment]
This is not caused by an ISP provider.
I checked this by doing a speedtest at the same time on another computer at home.
Also checked with my ISP. Nothing wrong with my line.
When I only use FS2020 my speed stays at 900+ Mbps.

I know it's hard to track something without any further info or logs (I checked ningx) but there's defintely something going on. Maybe an overload on my home network. I don't know.
